watertap/costing/units/electrodialysis.py

```diff
@@ -13,41 +13,23 @@
 import pyomo.environ as pyo
 from ..util import (
     register_costing_parameter_block,
+    cost_rectifier,
     make_capital_cost_var,
     make_fixed_operating_cost_var,
 )
 
 
 def build_electrodialysis_cost_param_block(blk):
 
-    blk.cem_membrane_cost = pyo.Var(
-        initialize=43,
-        doc="Cost of CEM membrane used in Electrodialysis ($/CEM/area)",
+    blk.membrane_capital_cost = pyo.Var(
+        initialize=150,
+        doc="Membrane and capitcal costs in [US$/m^2-membrane-area], referenced to Desalination 142 (2002) 267-286.",
         units=pyo.units.USD_2018 / (pyo.units.meter**2),
     )
-    blk.aem_membrane_cost = pyo.Var(
-        initialize=43,
-        doc="Cost of AEM membrane used in Electrodialysis ($/AEM/area)",
-        units=pyo.units.USD_2018 / (pyo.units.meter**2),
-    )
-    blk.flowspacer_cost = pyo.Var(
-        initialize=3,
-        doc="Cost of the spacers used in Electrodialysis ($/spacer/area)",
-        units=pyo.units.USD_2018 / (pyo.units.meter**2),
-    )
-    blk.factor_membrane_housing_replacement = pyo.Var(
+
+    blk.factor_membrane_equipment_replacement = pyo.Var(
```

Comment:
Similarly, should this be `stack` replacement factor instead of membrane?

Reply:
new names made.